configs/lc/hip/rocm-3.6-43-20200704_clang11.0.0.cmake

```diff
@@ -0,0 +1,13 @@
+#module load opt
```

Comment:
Add a comment explaining what these commented out lines are for.

Reply:
Done